src/Build/Evaluation/Evaluator.cs

```diff
@@ -2048,7 +2050,7 @@ private List<ProjectRootElement> ExpandAndLoadImports(string directoryOfImportin
                             }
 
                             ProjectErrorUtilities.ThrowInvalidProject(importLocationInProject, "ImportedProjectNotFound",
-								      importFileUnescaped, importExpressionEscaped);
+                                      importFileUnescaped, importExpressionEscaped);
```

Comment:
nit:
Why? Previously, it lined up better with the (
(I'd prefer it either get pushed out a little more or more of the space deleted. This feels like an awkward in-between.)